src/nimble.nim

```diff
@@ -815,6 +815,14 @@ Please specify a valid SPDX identifier.""",
       "MIT"
     )
 
+  if pkgLicense in ["GPL-2.0", "GPL-3.0", "LGPL-2.1", "LGPL-3.0", "AGPL-3.0"]:
+    let orLater = options.promptList(
+      "\"Or any later version\" clause?", ["Yes", "No"])
+    if orLater == "Yes":
+      pkgLicense.add("-or-later")
+    else:
+      pkgLicense.add("-only")
```

Comment:
Does the spec really want `-only` to be specified? I'd rather keep it at just `GPL-3.0` and assume that means "-only".

Reply:
Yes, it is officially required now https://spdx.org/licenses/

"GPL-3.0" without "only" was deprecated. It's either "only" or "or-later". I assume this was done to prevent ambiguity.